XCUITests/HomePageSettingsUITest.swift

```diff
@@ -209,6 +209,36 @@ class HomePageSettingsUITests: BaseTestCase {
             }
         }
     }
+    
+    func testChangeHomeSettingsLabel() {
+        //Go to New Tab settings and select Custom URL option
+        navigator.goto(HomeSettings)
+        waitForExistence(app.navigationBars["Home"])
+        //Enter a custom URL
+        enterWebPageAsHomepage(text: websiteUrl1)
+        app.textFields["HomeAsCustomURLTextField"].typeText(XCUIKeyboardKey.return.rawValue)
+        app.navigationBars.buttons.element(boundBy: 0).tap()
```

Comment:
We should try to go back to Settings screen using the graph, this is not very readable in a test...
If we are in `HomeSettings` screen and a[ back action](https://github.com/mozilla-mobile/firefox-ios/blob/master/XCUITests/FxScreenGraph.swift#L693) is defined, we would need to just use `navigator.goto(SettingsScreen)` 

I tried to do that that way and for some reason that back action does not work, we may have to define a new one or find the cause why it does work, but that should be the way...